### PR TITLE
Arybka2 oop

### PR DESCRIPTION
**Pull request by randrii**
- fixed some issues;
- reviewed code;
- updated structure of oop2 package (OOP part2 + Collections).

### CODE REVIEW

**Comment by randrii** [comment]
Added exceptions to oop2 project.

**Comment by olobanok** [comment]
please move your class to packages according to functionality (instead of keeping them in packages oop and oop2)

**Comment by randrii** [comment]
> please move your class to packages according to functionality (instead of keeping them in packages oop and oop2)
Done.

File: src/main/java/rybka/controller/Application.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        movies.add(new Movie("10:30, 12:30, 16:30", "The King's Speech", FilmCategory.DRAMA.getCategoryName(), 1.9, 2010));
+        movies.add(new Movie("13:30, 14:45, 15:45", "Despicable Me", FilmCategory.COMEDY.getCategoryName(), 1.6, 2010));
+        movies.add(new Movie("13:43, 15:23, 17:53", "Mega Piranha", FilmCategory.HORROR.getCategoryName(), 1.5, 2010));
+    }
+
+    public void showSessionList() {
+        writeFile("\nMovies list");
+        for (Movie movie : movies) {
+            System.out.println(movie.getSessions() + "\t" + movie.getTitle());
+            writeFile(movie.getSessions() + "\t" + movie.getTitle());
```
Comment (olobanok) [inline src/main/java/rybka/controller/Application.java:43]: why do you need to write data to file? :)

**Review by olobanok — APPROVED** [review]

File: src/main/java/rybka/controller/Application.java (hunk 1 above)
Comment (randrii) [reply src/main/java/rybka/controller/Application.java:43]: I used this for try-with-resources

Comment (randrii) [reply src/main/java/rybka/controller/Application.java:43]: > why do you need to write data to file? :)

It is not mistake, is it?

**Pull request merged**